tools/vulnerability-scan/wss-scan.sh

```diff
@@ -79,27 +89,23 @@ do
   then
     cd $repo_path
     echo "Scanning repo: "$gitBasePath$repo " Project: " $repo
-# Generate temporary `settings.gradle` file based on the name in `build.gradle`    
+        
```

Comment:
You removed the comment but leave extra space.

Reply:
Good points thanks!